analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConstructorOverridableCall.cs

```diff
@@ -55,4 +55,32 @@ public override void DoSomething()
             Console.WriteLine(this.foo.Length);
         }
     }
+
+    public class Class1
+    {
+        public virtual void DoSomething() { }
+    }
+
+    public class Class2 : Class1
+    {
+        public Class2()
+        {
+            DoSomething(); // FN
+        }
+    }
+
+    public class Class3 : Class2
+    {
+        public string Name { get; set; }
+
+        public Class3(string name)
+        {
+            Name = name;
+        }
+
+        public override void DoSomething()
+        {
+            base.DoSomething();
```

Comment:
To really show what the problem is here, instead of calling base.DoSomething, you should do something with Name property as it should be null at this point.